dist/tools/picotool/Makefile

```
@@ -0,0 +1,40 @@
+PKG_NAME=picotool
+PKG_URL=https://github.com/raspberrypi/picotool.git
+PKG_VERSION=de8ae5ac334e1126993f72a5c67949712fd1e1a4
+PKG_LICENSE=BSD-3-Clause
+
+# manually set some RIOT env vars, so this Makefile can be called stand-alone
+RIOTBASE ?= $(CURDIR)/../../..
+RIOTTOOLS ?= $(CURDIR)/..
+
+# Location of the picosdk package
+PICOSDK_DIR = ../../../picosdk
+PICO_SDK_PATH ?= $(PICOSDK_DIR)/picosdk
+
+USEPKG += picosdk
+
+PKG_SOURCE_DIR = $(CURDIR)/source
+PKG_BUILD_DIR = $(PKG_SOURCE_DIR)/build
+
+# Include pkg.mk after setting RIOTBASE otherwise it is not found when this
+# Makefile is called stand-alone
+include $(RIOTBASE)/pkg/pkg.mk
+
+# Create a dependency on the picosdk build
+$(CURDIR)/picotool: $(PKG_PREPARED) $(PICOSDK_DIR)/.prepared
+	@echo "[INFO] compiling picotool from source now"
+	@cd $(PKG_SOURCE_DIR) && cmake -B build -S . -DPICO_SDK_PATH=$(PICO_SDK_PATH) -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++
+	@cd $(PKG_BUILD_DIR) && env --unset=CC --unset=CXX $(MAKE)
+	@cp $(PKG_BUILD_DIR)/picotool $(CURDIR)/picotool
+
+# Create a phony target to ensure picosdk is built
+.PHONY: picosdk
+picosdk: $(PICOSDK_DIR)/.prepared
+
+$(PICOSDK_DIR)/.prepared:
+	$(MAKE) -C $(RIOTBASE)/dist/tools/picosdk
+
+all: $(CURDIR)/picotool
+
+clean::
+	rm -f picotool
```

```suggestion
	rm -f $(CURDIR)/picotool
```

Without the $(CURDIR), the path might be wrong depending on where the make command was started from:
```
cbuec@W11nMate:~/RIOTstuff/riot-canfix/RIOT$ make -C dist/tools/picotool clean
make: Entering directory '/home/cbuec/RIOTstuff/riot-canfix/RIOT/dist/tools/picotool'
rm -rf /home/cbuec/RIOTstuff/riot-canfix/RIOT/dist/tools/picotool/source/build
rm -f picotool
make: Leaving directory '/home/cbuec/RIOTstuff/riot-canfix/RIOT/dist/tools/picotool'
```

With `$(CURDIR)` it works as expected:
```
cbuec@W11nMate:~/RIOTstuff/riot-canfix/RIOT$ make -C dist/tools/picotool clean
make: Entering directory '/home/cbuec/RIOTstuff/riot-canfix/RIOT/dist/tools/picotool'
rm -rf /home/cbuec/RIOTstuff/riot-canfix/RIOT/dist/tools/picotool/source/build
rm -f /home/cbuec/RIOTstuff/riot-canfix/RIOT/dist/tools/picotool/picotool
make: Leaving directory '/home/cbuec/RIOTstuff/riot-canfix/RIOT/dist/tools/picotool'
```

Sorry, again, for the incorrect suggestion 😅

---

No worries thank you for your help